flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/DataMapManager.java

```diff
@@ -805,6 +833,126 @@ private void genSource(ExpressionNode expr, String[] names, int idx, StringBuild
         }
     }
 
+    private void genDeleteMappingSource(ExpressionNode expr, String[] names, int idx, List<TextEdit> textEdits) {
+        if (expr == null) {
+            return;
+        } else if (expr.kind() == SyntaxKind.MAPPING_CONSTRUCTOR) {
+            String name = names[idx];
+            MappingConstructorExpressionNode mappingCtrExpr = (MappingConstructorExpressionNode) expr;
```

Comment:
Shall we use `expr instanceof MappingConstructorExpressionNode mappingCtrExpr` and use that variable instead